crates/shared/src/current_block.rs

```diff
@@ -124,13 +140,38 @@ where
     }
 }
 
+#[derive(prometheus_metric_storage::MetricStorage)]
+pub struct Metrics {
+    /// How much a new block number differs from the current block number.
+    #[metric(buckets(0., 1., 2., 4., 8., 25.), labels("sign"))]
```

Comment:
I was going to suggest making sure the bucket sizes are useful but you already did that :+1: .